File: identicon/src/main/java/com/lelloman/identicon/GithubIdenticonView.java
package com.lelloman.identicon;

import android.content.Context;
import android.util.AttributeSet;

import com.lelloman.identicon.github.GithubIdenticonDrawable;

public class GithubIdenticonView extends IdenticonView {

	public GithubIdenticonView(Context context) {
		super(context);
	}

	public GithubIdenticonView(Context context, AttributeSet attrs) {
		super(context, attrs);
	}

	@Override
	protected IdenticonDrawable makeIdenticonDrawable(int width, int height, int hash) {
		return new GithubIdenticonDrawable(width, height, hash);
	}
}

File: identicon/src/main/java/com/lelloman/identicon/view/GithubIdenticonView.kt
package com.lelloman.identicon.view

import android.content.Context
import android.util.AttributeSet

import com.lelloman.identicon.drawable.GithubIdenticonDrawable

@Suppress("unused")
class GithubIdenticonView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null
) : IdenticonView(context, attrs) {
    override fun makeIdenticonDrawable(width: Int, height: Int, hash: ByteArray) =
        GithubIdenticonDrawable(width, height, hash)
}
